scripts/sandbox.sh

```diff
@@ -47,7 +47,7 @@ if [[ -d $SPRINTER_ENV_ROOT/sprinter/ ]]; then
     rm -rf $SPRINTER_ENV_ROOT/sprinter/
 fi
 echo "Installing global sprinter..."
-bin/sprinter install $SANDBOX_DIR/examples/sprinter.cfg || error "Issue installing global sprinter!"
+PYTHONPATH="" bin/sprinter install $SANDBOX_DIR/examples/sprinter.cfg || error "Issue installing global sprinter!"
```

Comment:
This was causing a headache for me. Brewed python suggests setting the PYTHONPATH to it's location. Virtualenv updates the PATH variable, but not PYTHONPATH. Clearing it out here, just for the sprinter installation.

With this issue, it is also advisable to call `sprinter update` commands as `PYTHONPATH="" sprinter update my_env`.